README.md

```diff
@@ -4,61 +4,80 @@ The indexer for the [Blobscan](https://github.com/Blobscan/blobscan) explorer im
 
 ## How it works?
 
-The indexer crawls the blockchain fetching information from both the Execution and Beacon clients. The data is processed and stored in a MongoDB database.
+The indexer crawls the beaconchain fetching information from both the Execution and Consensus clients. The data is collected and sent to the Blobscan API that will store it in a PostgreSQL database.
 
-## How to run locally?
+## Compile and run locally
 
 1. Install dependencies
 
 ```
 sudo apt install libssl-dev
 ```
 
-2. Git clone this repo.
+2. Git clone this repository.
 
 ```bash
 git clone https://github.com/Blobscan/blobscan-indexer.rs.git
+cd blobscan-indexer.rs
 ```
 
-3. Go to directory.
+3. Set the environment variables.
+
+**blobscan-indexer** interacts with other services (such as the execution and consensus clients). In a system where the defaults are not correct, they can be configured
+by using environment variables or by creating a `.env` file. You can use the `.env.example` file as a reference.
 
 ```bash
-cd blobscan-indexer.rs
+echo "SECRET_KEY=$(openssl rand -base64 32)" > .env
 ```
 
-4. Set the environment variables.
-
-Check the table below.
+For more information about available variables check out [the table below](#environment-variables).
 
-5. Run the indexer.
+4. Run the indexer.
 
 ```bash
 cargo run
 ```
 
-6. (Optional) Run the indexer using docker.
+5. Build a release
+
+```bash
+cargo build -r
+```
+
+## Docker images
 
-If you prefer to use docker we have created an image for the indexer which is available at [blossomlabs/blobscan-indexer](https://hub.docker.com/repository/docker/blossomlabs/blobscan-indexer/general).
+For convenience, we also provide docker images for blobscan-indexer.
+
+Running with defaults
 
 ```bash
 docker run --rm blossomlabs/blobscan-indexer:master
 ```
 
-## Environment variables
+Using environment variables
+
+```bash
+docker run -e BLOBSCAN_API_ENDPOINT=http://blobscan-api:3001 -e BEACON_NODE_RPC=http://beacon:3500 -e EXECUTION_NODE_URL=http://execution:8545 --rm blossomlabs/blobscan-indexer:master
```

Comment:
I think we are repeating the `--rm blossomlabs/blobscan-indexer:master` section twice.

Reply:
why? these two commands are equivalent ways of running the container with environment variables

Reply:
I've added a comment for the second command. Let me know if it's clearer now!